packages/grafana-ui/src/components/Button/ToolbarButton.tsx

```diff
@@ -21,24 +22,28 @@ export interface Props extends HTMLAttributes<HTMLButtonElement> {
   /** reduces padding to xs */
   narrow?: boolean;
   /** variant */
-  variant?: ButtonVariant;
-  /** Changes border color to orange */
-  active?: boolean;
+  variant?: ToolbarButtonVariant;
+  /** Hide any children and only show icon */
+  iconOnly?: boolean;
```

Comment:
this is a utility prop that makes the ResponsiveButton obsolete, a nice way to toggle rendering the children (text) via bool property